### DISCUSSION TITLE
Shelly 3EM mit SAE verbinden ?

### DISCUSSION THREAD

**Discussion by DrMabuse100** [camueller/SmartApplianceEnabler#323] [Installation und Betrieb]
Hallo,
breuchte einen kleinen TIP bitte ob es möglich ist die leistungwerte vom Shelly 3EM auf den SAE zu bringen ?

Dankeschön
Freundlicher Gruß

**Comment by CHB68** [comment]
das sollte doch gehen. Unter  Code steht:
kompatible Produkte
[Shelly Plug/3EM/4PM/Plus 1PM]
